src/hiero_sdk_python/consensus/topic_message.py

```
@@ -1,14 +1,12 @@
-"""
-This module provides the `TopicMessage` and `TopicMessageChunk` classes for handling
-Hedera Consensus Service topic messages using the Hiero SDK.
-"""
-
 from datetime import datetime
-from typing import Optional, List, Union, Dict
-
-from hiero_sdk_python.timestamp import Timestamp
+from typing import Optional, List, Union
```

Ruff (0.15.1)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)